### DISCUSSION TITLE
Support Google Drive Backup

### DISCUSSION THREAD

**Discussion by Nailik** [graphefruit/Beanconqueror#264] [Ideas]
Hi,
i would be very happy if google drive backup would be supported.
I often like to test some custom roms and whenever i forget to export and safe the backup (like today) i loose all my data in the app, that's pretty frustrating.

Would be very happy if you can consider this for some future updates!

**Comment by graphefruit** [comment]
Hi @Nailik,
sorry for the late response.
I've researched but there is no good plugin for this right now.
Therefore the return on invest is here is a bit to high.
I rather would like to support a user login and sync data to a backend completly.

Greetings
Lars

**Reply by Nailik** [reply]
I saw that there is an auto export option available.
It's not very well documented, because i'm not sure if it is on by default?

Setting a custom path for auto export would also help.
Because you could use a remote folder, for example the synology app creates a system folder that automatically stores files on your nas.

**Reply by graphefruit** [reply]
The auto export was built out of necessity, because sometimes users had data loss.
I had done many fixes and it looks like that the data loss was prevent since then.

The auto export just triggers the download of the .JSON-File but not for the images, and they are sent to the normal download folder.

After this app is built in hybrid framework, some things are not perfect possible like supporting to write or read from a SD-Card.
I had many struggles with the file systems on Android devices, so giving a custom path would be a solution, but like I said, there is no export of images on this auto export.
